assets/css/sensei-course-theme/lesson-title.scss

```diff
@@ -0,0 +1,8 @@
+.sensei-course-theme-lesson-title {
+	font-size: 40px;
+	line-height: 50px;
+	font-weight: 400;
+	margin-top: 0;
+	margin-bottom: 15px;
+	color: #1E1E1E;
```

Comment:
I think it's the default color of the whole content. Maybe we could just not define it here. WDYT? Also, about the `line-height`, maybe it would be interesting to set something relative, like `1.2` or other value that makes it like the layouts. Just to make the maintenance easier when changing font-sizes (same about the line-height for the title style). :)

Reply:
Of course! Done. 👍